taskcat/stacker.py

```diff
@@ -1693,6 +1699,32 @@ def check_yaml(self, yamlin, quite=None, strict=None):
             return False
         return True
 
+    def check_cfnyaml(self, yamlin, quite=None, strict=None):
+        """
+        This function validates the given Cloudforamtion YAML.
+
+        :param yamlin: CFNYaml object to be validated
+        :param quite: Optional value, if set True suppress verbose output
+        :param strict: Optional value, Display errors and exit
+
+        :return: TRUE if given yaml is valid, FALSE otherwise.
+        """
+        try:
+            loader = cfnlint.decode.cfn_yaml.MarkedLoader(yamlin, None)
+            loader.add_multi_constructor('!', cfnlint.decode.cfn_yaml.multi_constructor)
+            if self.verbose:
+                if not quite:
+                    print(loader.get_single_data())
+        except Exception as e:
+            if strict:
+                print(E + str(e))
+                sys.exit(1)
```

Comment:
Should we exit in one case and return false in another?

Reply:
Can add similar to  this https://github.com/aws-quickstart/taskcat/blob/master/taskcat/stacker.py#L1616-L1619 once we get familiar with how syntax checks are

Reply:
👍